docs/docs/configuration/auth.md

```diff
@@ -237,7 +237,7 @@ you should define the 'keycloak-group' value to /admin.
     --client-id=<your client's id>
     --client-secret=<your client's secret>
     --redirect-url=https://internal.yourcompany.com/oauth2/callback
-    --oidc-issuer-url=https://<keycloak host>/auth/realms/<your realm>
+    --oidc-issuer-url=https://<keycloak host>/realms/<your realm> // For Keycloak versions <17: --oidc-issuer-url=https://<keycloak host>/auth/realms/<your realm>
```

Comment:
```suggestion
    --oidc-issuer-url=https://<keycloak host>/realms/<your realm> // For Keycloak versions > 17: --oidc-issuer-url=https://<keycloak host>/auth/realms/<your realm>
```

please you the greater than sign

Reply:
As it currently is the `<` is correct. My intention was that most people can ignore the comment while those that still run a Keycloak version smaller than 17 see the first part and then find the correct example for their old version after the colon.

Reply:
Okay you are right, I was thinking the other way around, that the statement "For Keycloak versions smaller than 17:" was intended for the left side and not the right but now I see your point.